client/Cargo.toml

```diff
@@ -15,13 +15,19 @@ categories = ["api-bindings"]
 mime = { version = "0.3.17", default-features = false }
 prost = { version = "0.12", default-features = false, features = ["std"] }
 prost-types = { version = "0.12", default-features = false }
-reqwest = { version = "0.11", features = ["json", "rustls-tls"] }
+reqwest = { version = "0.12", features = ["json", "rustls-tls"] }
 serde = { version = "1.0.219", default-features = false, features = ["std", "derive"] }
 serde_json = { version = "1.0.140", default-features = false, features = ["std"] }
 thiserror = { version = "2.0.12", default-features = false }
 tokio = { version = "1.44.2", default-features = false }
 turnkey_api_key_stamper = { path = "../api_key_stamper", version = "0.0.2" }
 
+[features]
+reqwest_http2 = ["reqwest/http2", "reqwest/native-tls-vendored", "reqwest/native-tls-alpn"]
```

Comment:
I'd feel pretty good including `request/http2` in the list of  features included in line 18.

On the other hand, `reqwest/native-tls-vendored` and `reqwest/native-tls-alpn` are more of a niche thing? Could group them under a `request_native_tls` perhaps?

Reply:
Done.